src/code/z_eff_ss_dead.c

```diff
@@ -28,14 +28,14 @@ void func_80026230(GlobalContext* globalCtx, Color_RGBA8* color, s16 arg2, s16 a
     CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_eff_ss_dead.c", 129);
 }
 
-void func_80026400(GlobalContext* globalCtx, Color_RGBA8* color, s16 arg2, s16 arg3) {
+void EffectSsDead_SetCloseFogColorOpa(GlobalContext* globalCtx, Color_RGBA8* color, s16 timeLeft, s16 timeTotal) {
```

Comment:
About the `EffectSsDead` prefix I can change it sure but to what?

what do they set more than just fog? I think the `SetCloseFogColorOpa`/Xlu part is descriptive enough

I used `timer` because the function is passed, well, a timer value, and it's used like a typical time in a cos() wave I thought (`Math_CosS((0x4000 / timeTotal) * timeLeft)`), but I could use `progressCur`/`progressMax` if you prefer that?

Reply:
never occured to me that this file is prefixed with `effect_ss` when its not a softsprite effect lol

Anyway, we call this thing `ColorFilter` throughout the codebase, and abstract the fact that its actually achieved using fog to the user. Cause fog in that context doesnt make all too much sense.
Maybe we dont want to abstract it here, and only abstract it in z_actor. I dont have a solution to any of this naming, just putting it out there

Reply:
My point was that they set more than just the color itself. And they seem to have different implementations in the file, so we should be careful not to give names that aren't compatible with the other functions.

Anyway, at this point I'd say we should leave those functions unnamed for now since it's the only thing holding the PR afaict. Regarding the arguments, I'm not familiar with the math behind time defined like that in a cos wave, but we can probably go with these names if you think it's right.